.travis.yml

```diff
@@ -0,0 +1,90 @@
+branches:
+ only:
```

Comment:
probably this needs to be removed

Reply:
yes, we can either comment this out to build all (future) branches or exchange continuous-integration by master. Wasn't sure what we want so I left it in for now.